site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -242,6 +242,20 @@ populates chrome.runtime.lastError is not unchecked.
   </web-tab>
 </web-tabs>
 
+### Pass a message to a selected tab's content script
```

Comment:
Can we add a [heading ID](https://developer.chrome.com/docs/handbook/extensions/#header-ids) to this section?

Reply:
@AmySteam Done! I added IDs to a few that were missing them too. Let me know if I should remove any, or if there is a better naming scheme for them I should follow!